docs/how-to-guides/save-a-snippet/save-a-snippet.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Save a code snippet and add additional context with AI using the Pieces Desktop app
+description: Learn how to save a code snippet in Pieces for Developers
+---
+
+Pieces allows you to save a piece of code, or code snippet, into the Pieces app, where additional context is added using AI.
+
+This is a powerful way to keep track of the code that you may need to refer back to later, or to share with your team. Instead of copying code snippets into a notes app where they are easy to lose, or writing them in a physical notebook where you cannot search for them, Pieces is designed to make code snippets not only easily searchable, but augmented using AI to provide additional context and information, such as automatically tagging the snippet, and providing relevant links to documentation.
+
+This guide will walk you through the process for saving code snippets in the Pieces Desktop app, and have additional context added to your snippet automatically by the Pieces AI capabilities.
+
+## Prerequisites
+
+To save a code snippet, you will need:
+
+- [Pieces installed](/installation-getting-started/what-am-i-installing) on your machine
+- A code snippet that you want to save. This can be a code file, code copied to your clipboard, or even a picture of code, for example a photo from a lecture.
+
+## Steps
+
+To save a snippet, you will take the following steps:
+
+1. [Open Pieces Desktop](#open-pieces-desktop)
+1. [Add the code snippet](#add-the-code-snippet)
+1. [Review the context added to the snippet](#review-the-context-added-to-the-snippet)
+
+### Open Pieces Desktop
+
+To save a snippet, launch Pieces Desktop.
+
+You can either:
+- Click on Pieces OS in your menu bar or system tray, and select **Launch Pieces for Developers**
+- Run Pieces from your Windows Start menu, macOS Applications folder, or Linux Applications menu
+
+Once Pieces is open, select the **Saved Materials** tab from the left-hand menu.
+
+<div class="container">
+    <div class="row">
+        <div class="col col--8">
+            ![The saved materials tab](saved-materials-tab.webp)
+        </div>
+    </div>
+</div>
+
+### Add the code snippet
```

Comment:
@jimbobbennett would love to know your thoughts: 

We can add a small code snippet here in the how-to-guide as well for quick access and ask the user to copy that snippet and paste it, if they don't have a snippet handy.

Reply:
Good idea - let me add that.

Reply:
Added - both code inline and a downloadable file